core/templates/pages/error-pages/error-iframed.mainpage.html

```diff
@@ -39,7 +39,7 @@
     <div ng-controller="Base">
       <base-content>
         <content>
-          <div role="main" style="margin: 0 auto; position: relative;">
+          <div role="main" class="m-0 position-relative">
```

Comment:
What is m-0? Is it `margin: 0` everywhere? Isn't this different from `margin: 0 auto`? Did you try this on different-width viewports?

Reply:
Fixed